go.mod

```diff
@@ -269,7 +269,7 @@ require (
 	github.com/Azure/go-autorest/autorest/date v0.3.0 // indirect
 	github.com/Azure/go-autorest/logger v0.2.1 // indirect
 	github.com/Azure/go-autorest/tracing v0.6.0 // indirect
-	github.com/BurntSushi/toml v1.2.1 // indirect
+	github.com/BurntSushi/toml v1.3.2 // indirect
```

Comment:
If you are just upgrading the linter, why it is upgrading a bunch of deps in `go.mod`?

Reply:
Thank you for pointing this out, my bad. This was the result of running `inv install-tools` with a defined `go.work` file. Running the command with `GOWORK=off` correctly detected the packages and did not bump root deps